pygn_server.py

```diff
@@ -214,6 +214,8 @@ def generate_argparser():
 
     atexit.register(cleanup)
 
+    print(f'Server started.')
```

Comment:
Should this be to STDERR?

Reply:
It could be, but why would it be?

Reply:
Oh I see now that you are watching for this message at start time.  All good.